modules/ocf_www/manifests/logging.pp

```diff
@@ -7,6 +7,29 @@
     require => Package['nfs-kernel-server'],
   }
 
+  # log outbound requests
+  firewall_multi {
+    '101 log outbound request on death (v4)':
```

Comment:
this is not nice (I'd like to use firewall46) but ip6tables will complain if you give it ipv4 and vice versa

Reply:
This could be good as a code comment, but anyway thanks for mentioning it here, that was useful

Reply:
You can at least use a `default` resource to cut down on repetition though.

Reply:
@kpengboy++, or map on an array with the two destinations

Reply:
`default` is really neat, thanks